SCRIPTS/EVENT/ASUA;LICENSES!CULTIVATOR!~!LICENSE.js

```diff
@@ -1 +1,14 @@
-//John Towell  Put code here
+/*===========================================
+Title: ASUA;LICENSES!CULTIVATOR!~!LICENSE
+Purpose: Sends license to CAT when status becomes 'Inactive'
+Author: John Towell
+============================================== */
+
+try{
+    if(matches(appStatus, "Revoked", "Suspended")){

```

Comment:
what are all the statuses here? @lwacht @mjhart54 @mattjhart54 @grayquarter

Reply:
The CDFA folks will know, I don't sorry.